### ISSUE TITLE
Add key combinations for NVDA installer and portable NVDA to activate them

### ISSUE THREAD

**Issue by haruncetinkaya306** [nvaccess/nvda#17348]
### Is your feature request related to a problem? Please explain.
We may want to put the NVDA installer and its portable version on a USB stick and run it on a computer that does not have NVDA installed or even a screen reader. If we are lucky, we can just go to the desktop, that computer, from there we can boot into the USB and start NVDA, and then it will work as it should. Or we can open Narrator and run NVDA. But why would we need these?
### Explain the solution you want
Built-in screen readers have shortcut keys to start them. Although NVDA is not a built-in application, there should be shortcut keys to trigger NVDA to run. This is already available for installed NVDA, and the same should be done for portable and NVDA installer if possible. However, this should not have the same shortcut functions as installed NVDA, because we may want to run the installer or portable NVDA on an installed NVDA, and this would cause both to conflict.
### Explain the alternatives you are considering
None.
### Additional context
None.

**Comment by seanbudd** [comment]
We could do this by creating a shortcut to portable copies on creation, as well as the option from a portable copy to create a shortcut. In the mean time, you can do this manually by creating a shortcut for the application in windows, and editing the properties of the shortcut file to add a shortcut key.

I am yet to open an issue for this, but I would encourage someone to also add the ability to edit the installed NVDA shortcut key as part of this, and #17210

**Comment by haruncetinkaya306** [comment]
@seanbudd
Can't this be done for the installer as well?

**Comment by XLTechie** [comment]
It sounds like you are asking that portable copies on USB sticks, be given shortcut key start mechanisms, that will work on any computer you put them in.

If that's what you want, it can not be done.

You can manually make a shortcut on a target computer that will launch NVDA, or the installer, on a USB stick, but only after that copy already exists connected to that computer.

One computer can not force another to recognize a shortcut key by using a USB stick.

Windows used to be able to do something like that, and it lead to much malware.
(Including a rootkit by Sony, of all things)

**Comment by haruncetinkaya306** [comment]
@XLTechie
Yes, my request was for both portable and installer NVDAs. But if that can't be done, this topic can be closed. Thanks for your interest.

**Comment by XLTechie** [comment]
It can be done for portables, but only on a computer where the portable already is. If you move the portable to another computer, the shortcut will not be there unless you create it again.

**Comment by haruncetinkaya306** [comment]
@XLTechie
If you find it appropriate, I would like to see it implemented for portable NVDAs.

**Comment by seanbudd** [comment]
Closing - there's no user story that makes sense for creating a shortcut for the NVDA installer. There is a manual workaround using Windows available for those interested in creating custom shortcuts. An issue for creating custom shortcuts for portable/installed copies as described in https://github.com/nvaccess/nvda/issues/17348#issuecomment-2453765429 could be considered.
